web/src/Api/Config.elm

```diff
@@ -31,9 +31,10 @@ init maybeConfig =
         Just config ->
             config
 
+        -- Here the URL has been changed for local development
         Nothing ->
             Config
-                { restApiUrl = "https://api.stepstoadvancedreading.org"
+                { restApiUrl = "localhost:8000"
```

Comment:
We may want to change this back to make the default case the production API endpoint.